### PR TITLE
Book initialize app

### PR DESCRIPTION
**Pull request by maxkatkalov**

### CODE REVIEW

File: books_service/models.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    def count_books(self) -> int:
+        return Book.objects.filter(author=self).count()
+
+    def __str__(self) -> str:
+        return f"{self.first_name} {self.last_name}"
+
+
+class Book(models.Model):
+    COVER_CHOICES = (
+        ("hard", "Hard"),
```
Comment (lgSilay) [inline books_service/models.py:26]: i'm not sure, do we need duplicate this choises with differect characters?

File: books_service/models.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        ("soft", "Soft"),
+    )
+    title = models.CharField(max_length=255)
+    author = models.ForeignKey(
+        Author, on_delete=models.CASCADE, related_name="books"
+    )
+    cover = models.CharField(max_length=4, choices=COVER_CHOICES)
+    inventory = models.IntegerField()
```
Comment (lgSilay) [inline books_service/models.py]: Please change inventory field to Positive.IntergerFiled

File: books_service/models.py (hunk 1 above)
Comment (maxkatkalov) [reply books_service/models.py:26]: Not a duplication, read [here](https://docs.djangoproject.com/en/4.2/ref/models/fields/#choices)

File: books_service/models.py (hunk 2 above)
Comment (maxkatkalov) [reply books_service/models.py]: added with validation

**Review by lgSilay — APPROVED** [review]

**Review by innyshka — APPROVED** [review]

**Review by denys-source — APPROVED** [review]

**Comment by maxkatkalov** [comment]
[1a. Initialize books app](https://trello.com/c/N8zUxKik/14-1a-initialize-books-app)

**Pull request merged**